src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import core.basesyntax.service.ColorSupplier;
+
+public abstract class Figure implements CalculatorArea, Drawable {
+    protected static final int MAX_VALUE = 100;
+    protected String color = new ColorSupplier().getRandomColor();
```

Comment:
```suggestion
    private String color;
```
Lets add getter and setter for color